### DISCUSSION TITLE
Is Codespaces unstable for you too?

### DISCUSSION THREAD

**Discussion by behrangsa** [community/community#45609] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Continuously connects and disconnects.

**Comment by jge162** [comment]
@behrangsa Your codespace may not have enough memory or CPU allocated to it, which can cause it to run slowly.
Latency or other network-related issues can slow down the performance of a codespace. If the host machine on which the codespace is running is experiencing high resource usage from other processes, this can cause the codespace to run slowly. The code being run in the codespace may have performance issues or bugs that are causing it to run slowly.

You can also submit a [bug report](https://support.github.com/contact/bug-report) hope this helps.

**Reply by behrangsa** [reply]
> Your codespace may not have enough memory or CPU allocated to it

I tried up to 8 cores and 16GB RAM but the problem persisted. I stopped working for a few hours and came back and it looks stable again.

I tested my Internet speed/latency too when this issue was occurring and everything seemed to be normal.

Anyway, looks like it was a hiccup.

**Reply by jge162** [reply]
Glad to hear you got it working. Sounds like it was a high resource issue. Based on your response. Take care.
